### DISCUSSION TITLE
websocket 4.9.1 compatibility not loading.

### DISCUSSION THREAD

**Discussion by SerTokes** [Palakis/obs-websocket#994] [Q&A]
The title is pretty self explanatory here, but I'll offer more :

I updated to OBS v28 and with that update, the websocket was updated to 5.0.x

Now, here I am, with all of these barraider tools that no longer function, because they are built on 4.9.1

I found the compatibility dll, installed it per directions, and OBS very passively tells me that the plugin failed to load.

So here I am, with a bunch of non-functional buttons on my stream deck.

Anyone familiar with the problem and how to fix it?

**Comment by tt2468** [comment]
The compat download on the 5.0.1 release only works with v27, as stated in the release. We're working to make a dedicated release to support v28. Unfortunately, our CI has been having issues and it's taken more time than we thought it would.

**Reply by SerTokes** [reply]
Ahh, I see. I think I may have assumed that when I saw v27.2 or above that was up to and including v28.

My bad!

Thank you for the prompt repsonse, tho.  Super dope of you.

**Comment by LenhardErwan** [comment]
Hi, sorry to bother, but is there anything new?
